### PR TITLE
SystemAdmin feature scenarios addition

### PR DESCRIPTION
**Pull request by itsmekumari**
Few SystemAdmin feature design scenarios addition. Related framework level changes are in PR https://github.com/cdapio/cdap-e2e-tests/pull/215

### CODE REVIEW

File: cdap-e2e-tests/src/e2e-test/features/systemadmin/SysAdminDesignTime.feature [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+@Sysadmin
+Feature: Sysadmin - Validate system admin page design time scenarios
+
+  @Sysadmin @SysAdminRequired
+  Scenario:Validate user is able to create new system preferences and able to delete the added system preferences successfully
+    Given Open Datafusion Project to configure pipeline
+    When Open "System Admin" menu
+    Then Click on the Configuration link on the System admin page
+    Then Select "systemPreferences" option from Configuration page
+    Then Click on edit system preferences
```
Comment (radhikav1) [inline cdap-e2e-tests/src/e2e-test/features/systemadmin/SysAdminDesignTime.feature:28]: nit: you can use `Background` section  to define common steps that are to be run for all the scenarios in the feature.
Refer https://cucumber.io/docs/gherkin/reference/#background for more details and examples

**Review by radhikav1 — APPROVED** [review]
Please ensure that e2e test job on this PR runs fine with system admin helper https://github.com/cdapio/cdap-e2e-tests/pull/215 changes.

File: cdap-e2e-tests/src/e2e-test/features/systemadmin/SysAdminDesignTime.feature (hunk 1 above)
Comment (itsmekumari) [reply cdap-e2e-tests/src/e2e-test/features/systemadmin/SysAdminDesignTime.feature:28]: Added the Background section for the common steps in features.

**Comment by itsmekumari** [comment]
> Please ensure that e2e test job on this PR runs fine with system admin helper [cdapio/cdap-e2e-tests#215](https://github.com/cdapio/cdap-e2e-tests/pull/215) changes.

Yes, All the Sysadmin feature e2e tests are passed.

**Pull request merged**